.github/workflows/format.yml

```diff
@@ -99,7 +99,7 @@ jobs:
   clang_format:
     name: check format with clang-format
 
-    runs-on: ubuntu-20.04
+    runs-on: ubuntu-24.04
```

Comment:
It looks like the use of clang-format-12 needs to be updated as well--the clang_format action is currently failing with:
```
E: Unable to locate package clang-format-12
```